File: 23-09-26/2.c

#include <stdio.h>

int daysElapsed(int day, int month)
{
int days[] = {31, 28, 31, 30, 31, 30,
31, 31, 30, 31, 30, 31};

int total = day;  

for (int i = 0; i < month - 1; i++)  
{  
    total += days[i];  
}  

return total;

}

int main()
{
int day;
int month;

printf("Enter the day: ");
scanf("%d",&day);
printf("Enter the month: ");
scanf(" %d",&month);
int output = daysElapsed(day, month);  

printf("day = %d\n", day);  
printf("month = %d\n", month);  
printf("Days Elapsed = %d\n", output);  

return 0;

}

File: 23-09-26/2.py
import numpy as np

def daysElapsed(day, month):
    days = np.array([31, 28, 31, 30, 31, 30,
                     31, 31, 30, 31, 30, 31])

    total = day

    for i in range(month - 1):
        total += days[i]

    return total


day = input("Enter The Day: ")
month = input("Enter The Month: ")

output = daysElapsed(day, month)

print(f"day = {day}")
print(f"month = {month}")
print(f"Days Elapsed = {output}")
